packages/rs-drive-abci/src/platform_types/platform_state/v0/mod.rs

```diff
@@ -98,7 +99,7 @@ pub(super) struct PlatformStateForSavingV0 {
     /// Information about the last block
     pub last_committed_block_info: Option<ExtendedBlockInfo>,
     /// Current Version
-    pub current_protocol_version_in_consensus: ProtocolVersion,
+    pub current_protocol_version_in_consensus: ProtocolVersion, // TODO: Remove this
```

Comment:
why would you want to remove this?

Reply:
Oh, that's an artefact from our previous task when we were going to store platform state in trees